synapse/handlers/relations.py

```diff
@@ -210,8 +203,13 @@ async def get_bundled_aggregations(
             user_id: The user requesting the bundled aggregations.
 
         Returns:
-            A map of event ID to the bundled aggregation for the event. Not all
-            events may have bundled aggregations in the results.
+            A map of event ID to the bundled aggregation for the event.
```

Comment:
I'm a bit confused about the terminology here. The return type is `BundledAggregation`**`s`**, but this text suggests there can only be one?

Reply:
I think it was just a typo, it should be "bundled aggregations". There is only a single "bundle" per type of relation, but since there are multiple types of relations there are multiple "aggregations". 🤷 The terminology is all a bit confusing, yes.